waiter/src/waiter/scheduler.clj

```diff
@@ -482,9 +485,11 @@
                                         (conj scheduler-messages
                                               [:update-service-instances
                                                (assoc service-instance-info
-                                                 :service-id service-id
-                                                 :failed-instances all-failed-instances
-                                                 :scheduler-sync-time request-instances-time)])
+                                                      :failed-instances all-failed-instances
+                                                      :instances instances
+                                                      :scheduler-sync-time request-instances-time
+                                                      :service-id service-id
+                                                      :task-count task-count)])
```

Comment:
I was going to use both of these values (`instances` and `task-count`) to detect instances created due to scale-ups; however, since that isn't done yet, I'll remove these changes from this PR.

Reply:
I added back `instances` and `task-count` when I redid the instance tracking logic.